README.md

```diff
@@ -117,6 +117,12 @@ def main():
     deployer.finalize(deployments=deployments)
 ```
 
+To run a deployment locally:
+
+```bash
+ape run lynx deploy_infraction --network ethereum:local:test
```

Comment:
I guess this shouldn't be part of the readme, right?

Reply:
how come? I wasn't sure how to do it and @derekpierre explained, so I thought I'd add it in for next time :D

Reply:
The README is currently explaining how deployment scripts work generically, so an instruction that's specific to `lynx deploy_infraction` doesn't quite fit.

Reply:
There is an instruction in the `Deploy` section - https://github.com/nucypher/nucypher-contracts/blob/fdcb313c81102ab5d52e673dd3e9be80caf9f85b/README.md#4-deploy - that shows the command for using the local network and is more generic, and perhaps already achieves what you wanted to convey. Of course, you can reword/move things around for better clarity. 

Keeping it general is a good thing, and then optionally including specific examples if more clarity is needed.